src/vs/workbench/contrib/codeEditor/browser/outline/documentSymbolsTree.ts

```diff
@@ -106,15 +110,18 @@ export class DocumentSymbolDragAndDrop implements ITreeDragAndDrop<DocumentSymbo
 		}
 
 		const outlineElements = item instanceof OutlineElement ? [item] : Array.from(item.children.values());
-		const symbolsData = outlineElements.map(oe => ({
+
+		fillInSymbolsDragData(outlineElements.map(oe => ({
 			name: oe.symbol.name,
 			fsPath: resource.fsPath,
 			range: oe.symbol.range,
 			kind: oe.symbol.kind
-		}));
+		})), originalEvent);
```

Comment:
@benibenj Just a heads up: made a few changes here so I could reuse this logic for the inline anchors as well